src/Microsoft.AspNetCore.Http.Connections/Internal/HttpConnectionContext.cs

```diff
@@ -267,6 +267,9 @@ public void TickHeartbeat()
                 {
                     Log.ShuttingDownTransportAndApplication(_logger, TransportType);
 
+                    // Cancel any pending flushes from back pressure
+                    Application?.Output.CancelPendingFlush();
```

Comment:
Do we need this if we're going to be calling complete?

Reply:
Ah, I see where we use this. It does affect other code though. I guess the only place that we do this is websockets. Have you looked there to make sure we handle cancellation ok?

Reply:
I don't know where there is. Could you point me in the right direction.

I can say tests all pass 🙃

Reply:
https://github.com/aspnet/SignalR/blob/028d0b233e7ee515e66c6dc04ee5a3a42d2956a1/src/Microsoft.AspNetCore.Http.Connections/Internal/Transports/WebSocketsTransport.cs#L179